### DISCUSSION TITLE
How often does the website refresh?

### DISCUSSION THREAD

**Discussion by anderlli0053** [omaha-consulting/winstall#164] [Q&A] [answered]
How often does the website refresh?

If the package is added to [winget-pkgs](https://github.com/microsoft/winget-pkgs) today, will it be shown in https://winstall.app/ in a time span of 1 day?

**Comment by mherrmann** [comment]
Yes, but why do you ask?

**Comment by anderlli0053** [comment]
> Yes, but why do you ask?

I'm asking because, if I or someone else added new package from a [New Package Request] issue, and I was just curious as to how much time does it need to appear on the search engine.  

For example I added package request for NeoAxis Engine and also for Panda3D, someone created the manifest for both, created a pull request, pushed it, the CI system checked it and lastly accepted it and hours later neither of the 2 packages were listed on this search engine nor on **winget.run**

**Reply by mherrmann** [reply, marked as answer]
Do you mean these two Pull Requests?

 * https://github.com/microsoft/winget-pkgs/pull/155239
 * https://github.com/microsoft/winget-pkgs/pull/155242

They're still open. The apps will only appear on winstall.app once the PRs have been merged.

winget.run is no longer maintained. Compare the number of packages on their starting page - 4315 - to the one on winstall.app: 6100.

**Reply by anderlli0053** [reply]
> Do you mean these two Pull Requests?
> 
>  * https://github.com/microsoft/winget-pkgs/pull/155239
>  * https://github.com/microsoft/winget-pkgs/pull/155242
> 
> They're still open. The apps will only appear on winstall.app once the PRs have been merged.
> 
> winget.run is no longer maintained. Compare the number of packages on their starting page - 4315 - to the one on winstall.app: 6100. 

Why yes, thank you!
